packages/core-packages-documentation-generator/src/sdk-index-link-client.ts

```diff
@@ -71,7 +71,7 @@ export class SdkIndexLinkClientPlugin {
       const clientsCategory = group.categories.find((value) => value.title === "Clients");
 
       for (const child of clientsCategory.children || []) {
-        child.url = `/clients/${child.originalName}/`;
+        child.url = "https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest" + `/clients/${child.originalName}/`;
```

Comment:
```suggestion
        child.url = `https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/${child.originalName}/`;
```

Reply:
The basePath is defined in `typedoc.client.json`
Why do we need to define it here again?